src/strategies/types/types.ts

```diff
@@ -115,6 +116,15 @@ export namespace VerifyFunction {
       req?: Request,
     ): Promise<T | null>;
   }
+  export interface SamlFn<T = IAuthUser> extends GenericAuthFn<T> {
+    (
+      // accessToken: string,
+      // refreshToken: string,
```

Comment:
Why is this commented

Reply:
For SAML only 2 arguments are needed that's why it was commented.